app/src/main/java/org/jellyfin/androidtv/ui/preference/category/about.kt

```diff
@@ -1,28 +1,31 @@
 package org.jellyfin.androidtv.ui.preference.category
 
 import android.os.Build
+import org.jellyfin.androidtv.BuildConfig
 import org.jellyfin.androidtv.R
 import org.jellyfin.androidtv.ui.preference.dsl.OptionsScreen
 import org.jellyfin.androidtv.ui.preference.dsl.link
 import org.jellyfin.androidtv.ui.preference.screen.LicensesScreen
-import org.jellyfin.androidtv.util.Utils
 
 fun OptionsScreen.aboutCategory() = category {
 	setTitle(R.string.pref_about_title)
 
 	link {
-		setTitle(R.string.lbl_version)
-		content = Utils.getVersionString(context)
+		title = "Jellyfin app version"
```

Comment:
^ This is explicitly not translated to make it easier to find when we ask for the app version during troubleshooting/bug reports.